api/volume.go

```diff
@@ -127,6 +129,18 @@ func (s *Server) VolumeCreate(rw http.ResponseWriter, req *http.Request) error {
 		}
 	}
 
+	if volume.AccessMode == "" && volume.FromBackup != "" {
+		_, volumeName, _, err := backupstore.DecodeBackupURL(volume.FromBackup)
+		if err != nil {
+			return errors.Wrapf(err, "failed to decode backup url: %v", volume.FromBackup)
+		}
+		bs, err := s.m.ListBackupsForVolumeSorted(volumeName)
+		if err != nil || bs == nil {
```

Comment:
Since the backup labels update will be handled through mutating webhook. 

The error message and null of backup will be separate to handle in mutating function: 
 https://github.com/longhorn/longhorn-manager/blob/0d4593a5d1c666a1d1c8f6afeadffd2bde4a3c9f/webhook/resources/volume/mutator.go#L106-L115